box/5.0/modules/ROOT/pages/box-connector-studio.adoc

```diff
@@ -32,23 +32,36 @@ Anypoint Studio provides two ways to add the connector to your Studio project: f
 
 == Configure the Connector
 
-To use Box Connector in a Mule application, configure a global configuratuon element that can be used by as many Box Connectors as you require for your application.
+To use Box Connector in a Mule application, configure a global configuration element that can be used by as many Box Connectors as you require for your application.
+
+Enter the following information in the *General* tab of the *Global Element Properties* screen to configure JWT authentication:
+
+[%header,cols="30a,70a"]
+|===
+|Parameter|Description
+|Client id|The client identifier as assigned by the authorization server when the client application was registered.
+|Client secret|The client application's client secret.
+|Public key id| The ID of the public key configured in your Box instance to authenticate the application.
+|Entity id| The type of the entity ID: `EnterpriseId` or `UserId`.
+|Box sub type| The box sub type: `enterprise` or `user`.
```

Comment:
"subtype" is usually one word, so consider writing it that way in the description here and elsewhere.